clients/admin-ui/src/features/datastore-connections/system_portal_config/forms/ConnectorParametersForm.tsx

```diff
@@ -7,6 +7,7 @@ import { useLazyGetDatastoreConnectionStatusQuery } from "datastore-connections/
 import DSRCustomizationModal from "datastore-connections/system_portal_config/forms/DSRCustomizationForm/DSRCustomizationModal";
 import {
   AntButton as Button,
+  AntSelect,
```

Comment:
looks like this is now being imported twice. Can we just use the `Select` version?

Reply:
Yess! I left just the Select one now.